test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -433,19 +433,25 @@ def get_system_info(self, hostname=None):
             get_MemAvailable = False
             ram_search_count = 0
```

Comment:
we don't need this. once we are done with the for loop, we can check if get_MemAvailable is True or not. If False, we use MemFree, Buffers, and Cached values.

Reply:
If MemAvailable is there then we take system_ram value  from it and break for loop. So no need of get_MemAvailable variable.
 MemAvailable comes first, then Buffers and cached comes in file. if MemAvailable is not there then for loop will be stopped after getting all three required values. there is no point to continue for loop after getting all three values. So to stop for loop i used the Ram_search_count. I think ram_search_count is ok.

Reply:
ram_search_count = 0
for i in mem_info['out']:
      if "MemAvailable" in i:
           MemAvailable = float(i.split()[1]) / (2**20)
           break
     elif "MemFree" in i:
                    ram_search_count += 1
                    MemFree = float(i.split()[1]) / (2**20)
                elif "Buffers" in i:
                    ram_search_count += 1
                    Buffers = float(i.split()[1]) / (2**20)
                elif "Cached" in i:
                    ram_search_count += 1
                    Cached = float(i.split()[1]) / (2**20)
                if ram_search_count is 3:
                     self.system_ram = MemFree + Buffers + Cached
                     break